force-app/main/default/classes/triggerhandlers/CheckContainerCapacityHandler.cls

```diff
@@ -0,0 +1,113 @@
+public with sharing class CheckContainerCapacityHandler {
```

Comment:
Follow architecture of project, don't create new class for 2 methods, extend existing CarPartService class with your new 2 method. Then call these methods from CarPartTriggerHandler class on beforeInsert/before update. Check how it is implemented in CarService for example or in AccountService.

Reply:
delete this class and move ur methods to CarPartService